en/routing.texy

```diff
@@ -198,17 +198,36 @@ class RouterFactory
 }
 \--
 
-.[note]
-The `@return` annotation is important and required for getting the code to work.
-
 **File app/config/config.neon**:
 
 /--neon
 services:
-	routerFactory: App\RouterFactory
-	router: @routerFactory::createRouter
+	router: App\Router\RouterFactory::createRouter
 \--
 
+.[caution]
+Order of routes is important, because it try from top to bottom.  Basic rule is we **declare routes from specific do general**.
```

Comment:
„because it try from top to bottom“ je anglicky špatně; co třeba „because they are tried sequentially from the first one to the last one“?

„from specific do general“ -> „from the most specific to the most general“